### DISCUSSION TITLE
Mining Village Art

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#1584] [Art]
The Goblin Craftmanship all began in the famous mining village in Sunflower Isles. This island is filled with rare minerals and is where Goblin's honed their talent in smithing.

The mining village is an island that will be filled with mining gameplay, crafting and special events.

This is an art request for 'mining' and 'mountain based' styled assets for the game.

## Inspiration

The latest Sunnyside pack includes some mountain/mining assets that give a cut view of the mountain.

<img width="654" alt="Screen Shot 2022-10-31 at 1 55 23 pm" src="https://user-images.githubusercontent.com/11745561/198921436-9b680465-ef1e-4cdb-ae1e-63b43b24417c.png">

# Requests

There are a range of ways to contribute to the mining village and we encourage all types of artists to get involved ❤️ As always we lean on the community imagination and there are no limits to what you can provide. We have given a few examples of 'themed' items if you wish to use them.

## 1. Bumpkins

We will want a range of Bumpkin accessories and Bumpkin NPCs for the mining village.

You can either contribute wearable items for your Bumpkin or interesting NPCs for in-game.

Examples from the Bumpkin art competition ❤️ 

![miner](https://user-images.githubusercontent.com/11745561/198922126-c7443863-5ff3-4b24-9a37-32e37f43bc6d.png)

Ideas:

- Mining Outfit
- Cave explorer outfit
- Blacksmith outift

## 2. Music

Do you have some ideas that will fit the theme of a mine? We would love to hear your talent - more details please reach out to @Rofrtd

## 3. Mining Decorations + Rare Items

Designs for 'mining' based items and decorations. The mining area will consist of new minerals and craftable items.

When designing a decoration or rare item please provide an example of it placed inside of the game in the Land Expansion area.

Examples include:

- Statues
- Advanced mining tools
- Mine carts
- Old mining equipment

## 4. Land Decorations

Do you love making the map beautiful for all of our players? Now is your chance to submit decorations that we can use across the mining area.

Examples:

- Rare/Shiny minerals
- Mining Equipment
- Torches

# Submissions

- Fits game aesthetic and colours of the beach
- Original pieces of work
- Pixel sizing is 1x1px

_By contributing to the project you agree to the terms and conditions - https://docs.sunflower-land.com/support/terms-of-service. Your art can be used freely by the project and all community members_

When submitting art, please include the following:

- Your farm ID (we will make sure the artists get some goodies 🏆 )
- A minified version of your art piece
- An example of the art used inside of the game or on the Bumpkin

## Timeline

Mining Village will be launched post Christmas but we are looking at releasing early sneak peaks and rare items before than period 🚀 

As always, contributors will receive an airdrop of the game item they design (if it is tokenised) and also some goodies!

**Comment by LittleEins** [comment]
### Little Eins - part1
![exampllles-mining](https://user-images.githubusercontent.com/104184348/201709588-cc51bdee-0b52-4916-9516-95d930ee905a.png)
Minified Raw Files:
![stone1](https://user-images.githubusercontent.com/104184348/201709657-8dc5641c-445a-4d23-90ee-9ecf8ef441e9.png)
![stalagmite2](https://user-images.githubusercontent.com/104184348/201709650-7f11dea0-1bf1-4d0e-8106-fd1a6ebe2d22.png)
![stalagmite3](https://user-images.githubusercontent.com/104184348/201709654-f8724e6c-8477-412d-9aa0-6dda20d5cd2a.png)
![stalagmite](https://user-images.githubusercontent.com/104184348/201709658-d812a9e3-317b-4621-9f23-0f6615e91060.png)
![diamond-stone](https://user-images.githubusercontent.com/104184348/201709699-0627bea8-9daa-4392-92ff-ffa1835f369b.png)
![mineral1](https://user-images.githubusercontent.com/104184348/201709716-51e2a62b-ab20-4eba-a496-5d5fd0eafc38.png)
![ruby-node](https://user-images.githubusercontent.com/104184348/201709724-b622885b-e8b3-4e8c-942e-7327fcf9f042.png)

**Reply by adamhannigan** [reply]
The brown rocks are such a nice touch on this!

Love the red and blue ones as well - these are inspiring me for more resources 😮

**Comment by LittleEins** [comment]
### Little Eins - part 2
![exampllles-miningss](https://user-images.githubusercontent.com/104184348/206885525-8bc5d950-56ad-4d5c-ade9-0ca792c0a5f4.gif)

Minified File: 
With Goblin
![driller-goblin](https://user-images.githubusercontent.com/104184348/206885516-bbb5d6e9-9adb-4ae6-a59c-6d39a24c18b9.gif)
Driller only
![driller](https://user-images.githubusercontent.com/104184348/206885520-58613589-b081-4ffa-9b23-0f1f6f60c057.gif)

**Reply by adamhannigan** [reply]
Nice work! The subtle animation is a great fit for the aesthetic ❤️

**Reply by On1el** [reply]
![2](https://user-images.githubusercontent.com/104461938/206851890-a7c18e4a-90c6-4f1e-b19d-f3075e89ebd6.png)
![1](https://user-images.githubusercontent.com/104461938/206851891-785c88c1-7073-4180-9c3a-31979f4df2af.png)
I believe that in the original perspective we will also see the back of the caterpillar wheels

**Comment by Jot4Dev** [comment]
![minerador_02-export](https://user-images.githubusercontent.com/119550269/204948544-5ee6a169-7abb-4280-bebb-a6b817435666.gif)
![minerador_01](https://user-images.githubusercontent.com/119550269/204946550-395fd749-dd7d-41a5-aaba-8ee91478e3ed.gif)
![minerador_03](https://user-images.githubusercontent.com/119550269/204946588-b2ef5a20-b365-462d-873c-2dbfb46412dc.png)

**Comment by FarmerKuenster** [comment]
Some statue ideas I had for the Mining Village!
Also it would be cool if every land had a unique statue that represented the area (maybe with a small boost or special achievement that gets unlocked after you build/buy each statue)

![stone haven diamond statue](https://user-images.githubusercontent.com/104322749/205509519-bd3a8c79-bb67-47db-b9bf-26deb70e79ed.png)
![stone haven diamond statue-large](https://user-images.githubusercontent.com/104322749/205509524-e60ecfc2-2aeb-462b-8c6b-81221789fcb6.png)
![stone haven pickaxe statue](https://user-images.githubusercontent.com/104322749/205509530-204d0192-6819-4550-9d4c-79d735ef883e.png)
![stone haven pickaxe statue-large](https://user-images.githubusercontent.com/104322749/205509536-45b38af2-cfac-44b0-a691-0f3e5691eb60.png)

![stone haven-statues on land-export](https://user-images.githubusercontent.com/104322749/205509567-3ca67e02-c52c-4297-a5cc-56a961b93cb2.png)

discord: farmerkunester#1626
land ID: #32391
